exotica/include/exotica/KinematicTree.h

```diff
@@ -219,7 +219,7 @@ namespace exotica
             void ComputeJ(const KinematicFrame& frame, KDL::Jacobian& J);
 
             BASE_TYPE ModelBaseType;
-            BASE_TYPE ControlledBaseType;
+            BASE_TYPE ControlledBaseType = BASE_TYPE::FIXED;
```

Comment:
Does this compile?
```ControlledBaseType = BASE_TYPE::FIXED;``` should be done in the constructor or at initialization.

Reply:
It does, both on my machine and in Travis (green)

Reply:
ok. It's just non-const static members.